src/FNA3D_Driver_Vulkan.c

```diff
@@ -2154,7 +2160,7 @@ static uint8_t VULKAN_INTERNAL_ChooseSwapSurfaceFormat(
 	for (i = 0; i < availableFormatsLength; i += 1)
 	{
 		if (	availableFormats[i].format == desiredFormat &&
-			availableFormats[i].colorSpace == VK_COLOR_SPACE_SRGB_NONLINEAR_KHR	)
+			availableFormats[i].colorSpace == VK_COLOR_SPACE_SRGB_NONLINEAR_KHR)
```

Comment:
Tab got lost

Reply:
Tab looked like an error to me, what's the rule for it being there so I know where else to put it?

Reply:
It's a tab opposite to the one at the start of the parens, the one after the `if (`.

Now that I'm not on my phone...

```suggestion
			availableFormats[i].colorSpace == VK_COLOR_SPACE_SRGB_NONLINEAR_KHR	)
```